### ISSUE TITLE
No break can be triggered if no long breaks are configured

### ISSUE THREAD

**Issue by ptoews** [slgobinath/SafeEyes#431]
I've noticed that breaks weren't triggered anymore automatically. I couldn't even trigger them manually via the tray icon menu entry "Take a break now". Just nothing happened. 
I've checked `/var/log/syslog` and this is what is added when I tried to trigger it manually:

```
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]: Traceback (most recent call last):
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:   File "/usr/lib/python3/dist-packages/safeeyes/core.py", line 248, in __fire_start_break
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:     if not self.on_start_break.fire(self.break_queue.get_break()):
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:   File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 118, in get_break
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:     self.__current_break = self.next()
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:   File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 135, in next
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:     break_obj = self.__next_short()
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:   File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 178, in __next_short
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]:     longs[self.__current_long].time -= shorts[self.__current_short].time
Mar  6 10:15:57 philipp-MS safeeyes.desktop[12630]: TypeError: 'NoneType' object is not subscriptable
```

I had no long breaks configured, so I added one and the same happened. But after a restart it's working again. Still have to check if automatic triggering works now as well, but at least the "next break" info is now shown, which wasn't there before.

**Comment by daedroza** [comment]
I'm having this problem too. The author says this is fixed in 2.1.2 (https://github.com/slgobinath/SafeEyes/commit/bd4be002e12eb55b508b35fdf48ed606866ff0fb), but I still experience this.

`2021-03-28 09:33:22,405 [INFO]:[MainThread] Enable Safe Eyes
Exception in thread WorkThread:
Traceback (most recent call last):
  File "/usr/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/lib/python3/dist-packages/safeeyes/core.py", line 181, in __scheduler_job
    time_to_wait = self.break_queue.get_break().time * 60
  File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 118, in get_break
    self.__current_break = self.next()
  File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 135, in next
    break_obj = self.__next_short()
  File "/usr/lib/python3/dist-packages/safeeyes/model.py", line 178, in __next_short
    longs[self.__current_long].time -= shorts[self.__current_short].time
TypeError: 'NoneType' object is not subscriptable`

**Comment by slgobinath** [comment]
Oops my bad. The above fix was made after releasing 2.1.2. I will release a new version with the fix at my earliest.

**Comment by daedroza** [comment]
Thanks @slgobinath , I checked the commit history and indeed it was committed after 2.1.2 release.

**Comment by daedroza** [comment]
By any chance this is pushed @slgobinath

**Comment by slgobinath** [comment]
@daedroza sorry I couldn't find enough time to test the recent changes and release the fix. I will try my best to release it this weekend.

**Comment by slgobinath** [comment]
Released the fix in 2.1.3
